plugins/sbt/graphql-java-codegen-sbt-plugin/src/main/scala/io/github/dreamylost/graphql/codegen/GraphQLCodegenPlugin.scala

```diff
@@ -0,0 +1,198 @@
+package io.github.dreamylost.graphql.codegen
+
+import java.nio.file.{ Path, Paths }
+import java.util
+
+import com.kobylynskyi.graphql.codegen.{ GraphQLCodegen, GraphQLCodegenValidate }
+import com.kobylynskyi.graphql.codegen.model._
+import com.kobylynskyi.graphql.codegen.supplier.{ JsonMappingConfigSupplier, SchemaFinder }
+import sbt.internal.util.complete.DefaultParsers.spaceDelimited
+import sbt.{ AutoPlugin, Def, PluginTrigger, _ }
+import sbt.Keys.{ sLog, _ }
+
+import scala.collection.JavaConverters._
+
+/**
+ *
+ * @author liguobin@growingio.com
+ * @version 1.0,2020/7/15
+ */
+object GraphQLCodegenPlugin extends AutoPlugin {
+
+  //TODO if impl GraphQLCodegenConfiguration, can not use settingKey in override method
+
+  private val codegen = "2.2.1";
```

Comment:
Can you please extract this version to some properties file in order not to hardcode it in scala file?